command.go

```diff
@@ -120,5 +120,18 @@ func CreateCommand(commandType CommandType) (Command, error) {
 	return nil, ErrCommandNotRegistered
 }
 
+// ListCommands returns a list of all registered command types.
+func ListCommands() map[CommandType]func() Command {
```

Comment:
Rename to `RegisteredCommands` then LGTM!

Reply:
fixed